content/kubernetes/release-notes/k8s-6-2-8-10-2022-01.md

```diff
@@ -0,0 +1,128 @@
+---
+Title: Redis Enterprise for Kubernetes Release Notes 6.2.8-10 (January 2022)
+linktitle: 6.2.8-10 (January 2022)
+description: Support for Istio as ingress controller, K8s 1.22 (AKS, kOps, GKE), OpenShift 4.9. 
+weight: 66
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/release-notes/k8s-6-2-8-10-2022-01.md,
+    /kubernetes/release-notes/k8s-6-2-8-10-2022-01/,   ]
+---
+## Overview
+
+The Redis Enterprise [K8s 6.2.8-10](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.8-10) release provides support for the [Redis Enterprise Software release 6.2.8]({{<relref "/rs/release-notes/rs-6-2-8-october-2021.md">}}) and includes several enhancements and bug fixes.
+
+The key new features, bug fixes, and known limitations are described below.
+
+## Images
+
+This release includes the following container images:
+
+* **Redis Enterprise**: `redislabs/redis:6.2.8-53` or  `redislabs/redis:6.2.8-53.rhel7-openshift`
+* **Operator**: `redislabs/operator:6.2.8-10`
+* **Services Rigger**: `redislabs/k8s-controller:6.2.8-10` or `redislabs/services-manager:6.2.8-10` (on the Red Hat registry)
+
+## Feature improvements
+
+* Istio gateway/virtual services are supported as ingress controllers. Note that for Active-Active databases, the operator doesn't create ingress rules, and those should be manually configured (RED-64020)
```

Comment:
I updated the document in Google docs - mention here that Istio version 1.12.0 was tested.